src/ProgressOnderwijsUtils/Data/PocoDataReader.cs

```diff
@@ -22,10 +22,27 @@ public interface IOptionalObjectProjectionForDebugging
     public sealed class PocoDataReader<T> : DbDataReaderBase, IOptionalObjectListForDebugging
         where T : IReadImplicitly
     {
+        readonly struct Optional

```

Comment:
Is je endgame deze ook te gaan exposen?

Reply:
Nee, dit was puur bedoeld als lapmiddel totdat we zoiets exposed hebben, maar dat zou ik dan wel doordachter willen doen; de subset die hier nodig was stelt immers weinig voor.  Maar naast Maybe de Optional lijkt me ooit wel wenselijk, ja.